locale_update.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+
+##
+# This script is designed to update the l10n files and locale registration
+# for the standalone loop client. The source of the localization files
+# is https://github.com/mozilla/loop-client-l10n repository.
+# The loop client repo is assumed to be https://github.com/mozilla/loop-client.
+#
+# Run this script from the local version of loop-client. It assumes that a local
+# version of loop-client-l10n is in a parallel directory: ../loop-client-l10n.
+# Changes are then pushed back to loop-client.
+##
+
+import argparse
+import io
+import os
+import re
+import shutil
+import sys
+
+def main(l10n_src, l10n_dst, index_file_name):
+    print "deleting existing l10n content tree:", l10n_dst
+    shutil.rmtree(l10n_dst, ignore_errors=True)
+
+    print "updating l10n tree from", l10n_src
+
+    def create_locale(src_dir):
+        dst_dir = src_dir.replace('_', '-')
+        shutil.copytree(os.path.join(l10n_src, src_dir), os.path.join(l10n_dst, dst_dir))
+        return dst_dir
+    
+    locale_dirs = os.listdir(l10n_src)
+    locale_list = [create_locale(x) for x in locale_dirs if x[0] != "." and x != "templates"]
+
+    print "updating locales list in", index_file_name
+    with io.open(index_file_name, "r+") as index_file:
+        index_html = index_file.read()
+
+        new_content = re.sub(
+            '<meta name=(["|\'])locales\\1.*? content=(["|\']).*?\\2.*? />',
+            '<meta name="locales" content="' + ", ".join(locale_list) + '" />',
```

Comment:
Given we don't need to read the list, I think I'd prefer just to separate by comma, not comma and space.